packages/theme/components/HeaderNavigation.vue

```diff
@@ -9,6 +9,37 @@
       :link="localePath(`/c/categories/${category}`)"
     />
   </div>
+  <SfModal v-else-if="isCategoryTreeAvailable" :visible="isMobileMenuOpen">
+    <SfAccordion open="" :multiple="false" transition="" showChevron>
+      <SfAccordionItem
+        v-for="(cat, i) in categoryTree && categoryTree.items"
+        :key="i"
+        class="nav-item"
+        :header="cat.label"
+      >
+        <SfList>
+          <SfListItem>
+            <SfMenuItem
+              label="All"
+              class="sf-header-navigation-item__menu-item"
+              :link="localePath(`/c/categories/${cat.label.replace(/\s+/g, '-').toLowerCase()}`)"
```

Comment:
Let's not do such magic :) It's also possible that a category will have a slug that's different from its label. 
You should be able to rely on the `slug` property of a category. 
So the whole line would be like ```:link="localePath(`/c/${cat.slug}`)``` Same for nested categories.